src/overview/actions.js

```diff
@@ -31,8 +31,25 @@ export function init() {
 // Search for docs matching the current query, update the results
 export function refreshSearch({loadingIndicator = false}) {
     return async function (dispatch, getState) {
-        const { query, startDate, endDate } = ourState(getState())
-        const oldResult = ourState(getState()).searchResult
+        let {
+            query,
+            startDate: oldStartDate,
+            endDate: oldEndDate,
+            searchResult: oldResult,
+        } = ourState(getState())
+
+        const {
+            startDate: newStartDate,
+            endDate: newEndDate,
+            extractedQuery,
+        } = extractTimeFiltersFromQuery(query)
+
+        const startDate = newStartDate || oldStartDate
+        const endDate = newEndDate || oldEndDate
+        query = extractedQuery || query
+
+        if (oldStartDate !== newStartDate) dispatch(setStartDate({startDate}))
+        if (oldEndDate !== newEndDate) dispatch(setEndDate({endDate}))
```

Comment:
Ok great was just a misunderstanding. I thought we should also use this NLP stuff in search field on overview. No problem than.